bootstrap/api/transport.go

```diff
@@ -105,10 +105,17 @@ func MakeHandler(svc bootstrap.Service, reader bootstrap.ConfigReader) http.Hand
 }
 
 func decodeAddRequest(_ context.Context, r *http.Request) (interface{}, error) {
-	if r.Header.Get("Content-Type") != contentType {
-		return nil, errUnsupportedContentType
+	s := r.Header.Get("Content-Type")
```

Comment:
Hi @tony-ward what @anovakovic01 meant with [this comment](https://github.com/mainflux/mainflux/issues/639#issuecomment-471440902) was to change the oriignal line:
```
if r.Header.Get("Content-Type") != contentType {
```
with
```
if !strings.Contains(r.Header.Get("Content-Type"), contentType) {
```

If we do it as in your implementation, to avoid code duplication, it would be better if we make a separate package which will parse the content type header. IMHO, at the moment, there is no need for that as we only work with utf-8 encoded strings and don't care about other character encodings.
If in the future we decide that we will support different encodings we can parse the header and possibly convert the strings to utf-8

Reply:
Great feedback, @chombium. I agree with you. @tony-ward There is no need for this much code, the simple snippet that @chombium suggested is good enough.